src/Mock.spec.ts

```diff
@@ -633,7 +633,8 @@ describe('vitest-mock-extended', () => {
   describe('mock Date', () => {
     test('should call built-in date functions', () => {
       type objWithDate = { date: Date }
-      const mockObj = mock<objWithDate>({ date: new Date('2000-01-15') })
+      const unixTimestamp = 947980800000 // Jan 15, 2000 at midnight UTC
+      const mockObj = mock<objWithDate>({ date: new Date(unixTimestamp) })
```

Comment:
I guess you could try, which also seems to work for me locally
```suggestion
      const unixTimestamp = Date.parse("15 Jan 2000 00:00:00 UTC"))
      const mockObj = mock<objWithDate>({ date: new Date(unixTimestamp) })
```